tests/utils/ssh/ssh.go

```diff
@@ -45,3 +45,13 @@ func InvokeCommandLocally(cmd string) string {
 	}
 	return strings.TrimSpace(string(out[:]))
 }
+
+//ExecCmd method takes command and host and calls InvokeCommand
+//and then returns the output after converting to string
+func ExecCmd(hostName string, cmd string) (string, error) {
```

Comment:
with #1294 there is an ask for getting rid of this util. can you please take care as part of this change?